eng/pipelines/templates/stages/archetype-net-release.yml

```diff
@@ -23,12 +23,12 @@ stages:
                   artifact: ${{parameters.ArtifactName}}
                   timeoutInMinutes: 5
 
-                - ${{ each artifact in parameters.Artifacts }}:
-                  - pwsh: |
-                      New-Item -Type Directory -Name staging -Path $(Pipeline.Workspace) -Force
-                      Copy-Item $(Pipeline.Workspace)/${{parameters.ArtifactName}}/${{artifact.name}}.[0-9]*.[0-9]*.[0-9]* $(Pipeline.Workspace)/staging
-                      Get-ChildItem $(Pipeline.Workspace)/staging
-                    displayName: Copying ${{artifact.name}} to staging directory
+                - pwsh: |
+                    New-Item -Type Directory -Name staging -Path $(Pipeline.Workspace) -Force
+                    Copy-Item $(Pipeline.Workspace)/${{parameters.ArtifactName}}/* $(Pipeline.Workspace)/staging -recurse
```

Comment:
Why do we need to copy anything? Part of the point of having these be separated in the artifact itself is to avoid the need for copying and partitioning further.

Reply:
If for some reason the sizing steps still need this then we should do a copy using and include of *.nupkg

Reply:
Yes the signing steps requires that we copy out the artifacts because it potentially mutates the artifacts while signing them. Yes we can use the include *.nupkg

Reply:
What happens if we simply pass the entire path to the signing step? Also do we need to also exclude the *.symbols.nupkg?

Reply:
It is ok if the packages get signed in place as we are going to be re-publishing them into the packages-signed artifact and we want that to have exactly the same files and structure but just have signed files instead of unsigned. There should be no reason we need to have the unsigned packages as part of the signing job after we do the signing.

Reply:
If this is the case do we need to republish it as `packages-signed` since it will be the same as `packges` except for the presence of docs.